main/mariadb/APKBUILD

```diff
@@ -315,6 +320,15 @@ _server_utils() {
 }
 
 sha512sums="7702af8787973b15784dc8a89f9d56765725a420e2e68f1924b67c88c134c7d83d3249dddd16707fa9e5f05ed747f813f16feefbb477e999a6698b1dcfb5a88f  mariadb-10.2.15.tar.gz
+_plugin_rocksdb() {
+	pkgdesc="MariaDB plugin for RocksDB (MyRocks)"
+	url="https://mariadb.com/kb/en/library/myrocks/"
+	depends="$pkgname=$pkgver-r$pkgrel"
+	mkdir -p "$subpkgdir"/usr/lib/mariadb/plugin
+	mv "$pkgdir"/usr/lib/mariadb/plugin/ha_rocksdb.so \
+		 "$subpkgdir"/usr/lib/mariadb/plugin/ha_rocksdb.so
+}
+
```

Comment:
Ehm...?

Reply:
@jirutka Is that an attempt at constructive feedback? Or how should it be interpeted?

Reply:
I was meant as a notice for your to check what you did here, 'cause there's very obvious mistake - this code is _inside_ the value of the `sha512sums` variable, so quite obvious mistake. Unfortunately if you read it directly on the PR page and not the file page, there's not enough context to see it.